provider/awsprovider/awsprovider.go

```diff
@@ -36,6 +36,12 @@ func (p *AWSProvider) ConfigureRun(state *run.State) error {
 	}
 
 	state.RootURL = userData.RootURL
+	state.WorkerLocation = map[string]string{
+		"cloud":        "aws",
+		"instanceId":   iid_json.InstanceId,
+		"instanceType": iid_json.InstanceType,
```

Comment:
These two are about the instance, not its location, and should probably be omitted.